python/google/protobuf/internal/message_test.py

```diff
@@ -2589,5 +2589,35 @@ def testSucceedOversizeProto(self):
     q.ParseFromString(self.p_serialized)
     self.assertEqual(self.p.field.payload, q.field.payload)
 
+
+@unittest.skipIf(api_implementation.Type() != 'cpp',
+                 'explicit tests of the C++ implementation for PY27 and above')
```

Comment:
'explicit tests of the C++ implementation'

Reply:
Amended.